src/app/pages/home/prysmatic-home.page.component.html

```diff
@@ -150,11 +150,11 @@ <h4 class="card-subtitle text-grey-darkest text-sm">A full-featured client for t
                     </div>
                     <div class="w-full px-6 pt-4 pb-8">
                         <h2 class="text-2xl">
-                            <a href="https://prylabs.net" class="text-grey-darkest">Eth2 Mainnet Phase 0</a>
+                            <a href="https://launchpad.ethereum.org/" class="text-grey-darkest">Eth2 Mainnet Phase 0</a>
                         </h2>
                         <h4 class="text-grey-darkest text-sm">The official release of eth2</h4>
                         <p class="py-4">
-                            Scheduled to launch December 2020, eth2 is a publicly accessible network running multiple client implementations participating in Ethereum 2.0 consensus based on the <a class="text-primary" href="https://github.com/ethereum/eth2.0-specs" target="_blank">formal specification</a> for the protocol. The official and only validator deposit contract is <a target="_blank" class="text-primary" href="https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa">0x00000000219ab540356cbb839cbe05303d7705fa</a>.
+                            Launching December 1st 2020, eth2 is a publicly accessible network running multiple client implementations participating in Ethereum 2.0 consensus based on the <a class="text-primary" href="https://github.com/ethereum/eth2.0-specs" target="_blank">formal specification</a> for the protocol. The official and only validator deposit contract is <a target="_blank" class="text-primary" href="https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa">0x00000000219ab540356cbb839cbe05303d7705fa</a>.
```

Comment:
IMO, just changed this to past tense so we don't have to update it again in a few days. 

```suggestion
                            Launched December 1st 2020, eth2 is a publicly accessible network running multiple client implementations participating in Ethereum 2.0 consensus based on the <a class="text-primary" href="https://github.com/ethereum/eth2.0-specs" target="_blank">formal specification</a> for the protocol. The official and only validator deposit contract is <a target="_blank" class="text-primary" href="https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa">0x00000000219ab540356cbb839cbe05303d7705fa</a>.
```